android/src/main/java/com/reactlibrary/JudoReactNativeModule.kt

```diff
@@ -115,35 +115,44 @@ class JudoReactNativeModule internal constructor(context: ReactApplicationContex
 
     private fun getTransactionConfiguration(options: ReadableMap): Judo? {
```

Comment:
`Judo` cannot be optional, it'a a required parameter, if you can't build it, you throw an exception